i18n/en/docusaurus-plugin-content-docs/current/reference/units/layers/widgets.md

```diff
@@ -37,6 +37,7 @@ And if the first option is easy to implement, then everything becomes not so tri
 
 ```tsx title=widgets/header/ui.tsx
 import { SearchBar } from "features/search-bar";
+import { Navigation } from "features/navigation";
```

Comment:
Thx for your care! Could u please add same import on ru version of article?

Reply:
Yes I'll do it.